### ISSUE TITLE
Docker Container

### ISSUE THREAD

**Issue by jdineshbabu** [banago/simple-php-website#63]
Can we able to run this application inside the container? 
Also why this docker is running only under the folder /var/www/html/. could you please explain I'm new to docker

**Comment by banago** [comment]
Yes, you are able to run this in a docker container - just put the code in your working directory.
